Zend/zend_builtin_functions.c

```diff
@@ -1851,7 +1865,7 @@ static int copy_function_name(zval *zv, int num_args, va_list args, zend_hash_ke
 		char *disable_functions = INI_STR("disable_functions");
 
 		if ((*exclude_disabled == 1) && (disable_functions != NULL)) {
-			if (strstr(disable_functions, func->common.function_name->val) == NULL) {
+			if (!is_name_in_list(func->common.function_name, disable_functions)) {
```

Comment:
This looks really fragile, and I think it is still broken with regard to case-insensitivity. I'd suggest to check against `ZEND_FN(display_disabled_function)` instead.

Reply:
So simple! :) Thanks!